test/wsdl/elementref/bar1.xsd

```diff
@@ -5,34 +5,30 @@
     <xsd:element name="paymentRq">
         <xsd:complexType>
             <xsd:sequence>
-                <xsd:sequence>
```

Comment:
I just cleaned up to avoid redundancy. With the proposed changes, the nesting sequence is supported too. I can add it back.